.travis/matrix-runner.sh

```diff
@@ -19,6 +19,7 @@ if [ "${MATRIX_TYPE}" = "python" ]; then
     coverage run manage.py test --noinput --failfast --traceback --verbosity=2 --testrunner=$TESTRUNNER
 elif [ "${MATRIX_TYPE}" = "javascript" ]; then
     psql -c 'create database commcarehq' -U postgres
+    python manage.py sync_couch_views
```

Comment:
Doesn't `migrate` also sync the couch views? or is it that `migrate` need the views already synced?